package.json

```diff
@@ -20,6 +20,7 @@
     "tsnode": "ts-node --project tsconfig.examples.json -r tsconfig-paths/register",
     "preexample": "npm run build:main",
     "example": "node extra_docs/examples/javascript/node/node-example.js",
+    "example_nossl": "cross-env NODE_TLS_REJECT_UNAUTHORIZED=0 node extra_docs/examples/javascript/node/node-example.js",
```

Comment:
I think is up to the user to use `NODE_TLS_REJECT_UNAUTHORIZED`. This has to be explicitly indicated by user when running this script.